sherpa/astro/xspec/tests/test_xspec.py

```diff
@@ -805,3 +816,37 @@ def test_apec_redshift_parameter_is_case_insensitive():
     mdl.REDshift = 0.2
     assert mdl.redshift.val == 0.2
     assert mdl.Redshift.val == 0.2
+
+
+@requires_xspec
+def get_xspec_models_with_exp_cache():
+    """Return the XSPEC models that use the modelCacher1d_exp cache.
+
+    Currently, that is: Models that inherit from XSAbsorptionModel.
+    """
+    models = get_xspec_models()
+    return [m for m in models if issubclass(m, XSAbsorptionModel)]
+
+
+@requires_xspec
+@pytest.mark.parametrize("modelcls", get_xspec_models_with_exp_cache())
+def test_models_with_exp_cache(modelcls):
+    '''Tests that models with the modelCacher1d_exp work correctly.
+
+    Unlike modelCacher1d, does not evaluate the model for every
+    parameter value once, but modelCacher1d_exp assumes a certain
+    functional form and calcualtes model values by itself, this this
+    warrantes a test for every model using this cache.
```

Comment:
```suggestion
    warrants a test for every model using this cache.
```

Reply:
I think this paragraph could just be as simple as

    Test the models that use modelCacher1d_exp.